src/AvatarMixer.ts

```diff
@@ -114,8 +114,8 @@ class AvatarMixer extends AssignmentClient {
 
 
     /*@sdkdoc
-     *  Game loop update method that should be called multiple times per second to keep the avatar mixer up to date with user
-     *  client avatar state.
+     *  Game loop update method that needs to be called multiple times per second to keep the avatar mixer up to date with user
```

Comment:
Actually, on a second thought. How many times per second does this mean? 5? 30? 60?